packages/flutter_webrtc/tizen/project_def.prop

```diff
@@ -0,0 +1,29 @@
+# See https://docs.tizen.org/application/tizen-studio/native-tools/project-conversion
+# for details.
+
+APPNAME = flutter_webrtc_tizen_plugin
+type = sharedLib
+profile = common-5.5
+
+# Source files
+USER_SRCS += src/*.cc third_party/uuidxx/*.cc
+
+# User defines
+USER_DEFS =
+USER_UNDEFS =
+USER_CPP_DEFS = FLUTTER_PLUGIN_IMPL
+USER_CPP_UNDEFS =
+
+# Compiler flags
+USER_CFLAGS_MISC =
+USER_CPPFLAGS_MISC =
+
+# User includes
+USER_INC_DIRS = inc src third_party/libwebrtc/include third_party/uuidxx
+USER_INC_FILES =
+USER_CPP_INC_FILES =
+
+# Linker options
+USER_LIBS = webrtc
```

Comment:
The libs are more than 10MB and that's quite too large to be embedded in an app package. Did you optimize out symbols and etc. properly when building them?

Reply:
I check the build option , it is in release mode and the symbol_level = 0, it means no symbol.

Reply:
Hmm. Is there no more option to reduce the size?

Reply:
Ok，let me try.

Reply:
I have add some optimize compile options:
1. arm library size from 12.5M to 12M
2. arm64 library size from 16.5 to 15.8M
I will try more in the future.

Reply:
Thank you. It would be nice if the binary size could be more cut down before we put the binary into our source control. According to [this comment](https://groups.google.com/g/discuss-webrtc/c/illCSnZQJg4/m/HeNmE2T5CQAJ), the binary size on Android is around 4-7MB (at least back then in 2018) and I think the size should not be very different on Linux. If you don't currently know of any configuration option to get a smaller binary, let's leave it as it.

Reply:
I check the build option, currently linux not support optimize_for_size:
```bash
optimize_for_size = is_android || is_chromecast || is_fuchsia || is_ios
```